Godel.Microservice.CurrencyRates/MappingHelper/MappingProfile.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper;
+using Godel.Microservice.CurrencyRates.Business.Dto;
+using Godel.Microservice.CurrencyRates.Data.Entities;
+using Godel.Microservice.CurrencyRates.Model;
+
+namespace Godel.Microservice.CurrencyRates.MappingHelper
```

Comment:
please change name from MappingHelper to Mappings

Reply:
Was renamed from MappingHelper to Mappings